.env

```
@@ -0,0 +1,4 @@
+DATABASE_NAME=spring
+DATABASE_USER=user
+DATABASE_PASSWORD=secret
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_PASSWORD key should go before the DATABASE_USER key

(UnorderedKey)